config/global.go

```diff
@@ -0,0 +1,95 @@
+package config
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/coreos/etcd/client"
+	"golang.org/x/net/context"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var (
+	timeoutFixBase = time.Minute
+	ttlFixBase     = time.Second
+)
+
+// Global is the global configuration.
+type Global struct {
+	Debug   bool
+	Timeout time.Duration
```

Comment:
I think we may need to name these fields in a slightly more descriptive way as otherwise it will be hard to tell what are they used for in absence of a commanline help string, like perhaps this field should be called `CephTimeout` . Same will be true with ttl and may be we should call it `MasterAliveTTL` or similar?